### PR TITLE
add(module 1 & module 2): all steps and solutions 

### LINKED ISSUE #14: Create Katacoda Scenario for Module 1: General Overview of PF-Core
**content here:** https://docs.google.com/document/d/1Lq69PCWcesXcIzrLJLPgq6Ox3XQp8uu4HV2woMW9DGs/edit

### LINKED ISSUE #15: Create Katacoda Scenario for Module 2: BEM Naming System
**content here:** https://docs.google.com/document/d/1Lq69PCWcesXcIzrLJLPgq6Ox3XQp8uu4HV2woMW9DGs/edit

### PR DESCRIPTION
**Pull request by lareplog**
Added all the steps, files, and solutions for the first module. closes #14 and closes #15

### CODE REVIEW

File: html-css/module-1/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<div> test </div>>
```
[the remark this answers is not in the document]
Comment (dgutride) [reply html-css/module-1/index.html]: the index file is unnecessary - it should be deleted as you mention.

File: html-css/module-2/step4-instructions.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+`--pf-c-title--m-5xl--FontSize`
+
+k. Now create a declaration with the global variable you added as the value for your new custom property.
+`--pf-c-title--m-5xl--FontSize: var(--pf-global--FontSize--5xl);`
+l. Adding that declaration to the codepen, we should have.
+`.pf-c-title {
+  --pf-c-title--m-5xl--FontSize: var(--pf-global--FontSize--5xl);
+}`
+
+m. Next we need to write a CSS selector for the new 5xl variation. Knowing that the modifier .pf-m-5xl will apply to the title component, let's use the & in SCSS nested inside of .pf-c-title to create .pf-c-title.pf-m-5xl.
```
Comment (jenny-s51) [inline html-css/module-2/step4-instructions.md]: for some reason the code blocks aren't showing up in these last two steps (m. and n.) ... not sure why! they seem to look like all the others before it. maybe also try wrapping these CSS code blocks with ```

File: html-css/module-1/step1-answer.md [hunk 3]
```diff
@@ -1 +1,6 @@
-<div> code goes here </div>
\ No newline at end of file
+<button class="pf-c-button pf-m-primary" type="button">
```
[the remark this answers is not in the document]
Comment (lareplog) [reply html-css/module-1/step1-answer.md]: fixed all of them!!

File: html-css/module-2/step3-instructions.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+a. We are going to add a custom icon element to the label component and apply a margin-right of 8px (--pf-global--spacer--sm)
+
+b. Let's assume there is a common use case for adding an icon to the left of text in the label component when used in your application. Start out by adding a label component.
+<span class="pf-c-label">
+    My custom label
+</span>
+
+c. Then place an icon before the label text
+<span class="pf-c-label">
```
Comment (jenny-s51) [inline html-css/module-2/step3-instructions.md]: make sure you wrap these <spans> with ` too!

File: html-css/module-2/step3-instructions.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+As we have seen, in order to override variables its important to understand how BEM is used to name them. It’s just as important when creating variables.
+
+## Workshop: Add a BEM element and custom styles with CSS custom property to support an icon element in the Label component
+
+a. We are going to add a custom icon element to the label component and apply a margin-right of 8px (--pf-global--spacer--sm)
+
+b. Let's assume there is a common use case for adding an icon to the left of text in the label component when used in your application. Start out by adding a label component.
+<span class="pf-c-label">
```
Comment (jenny-s51) [inline html-css/module-2/step3-instructions.md]: make sure you wrap these <spans> with ` too!

File: html-css/module-2/step3-instructions.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+</span>
+
+c. Then place an icon before the label text
+<span class="pf-c-label">
+  <i class="fas fa-plus"></i>
+  My custom label
+</span>
+
+d. The label component only styles the label itself and the text inside. While this looks OK, we need to add space between the icon and the text beside it. To do this the BEM way, let's assign a BEM element class to the icon.
+<span class="pf-c-label">
```
Comment (jenny-s51) [inline html-css/module-2/step3-instructions.md]: make sure you wrap these <spans> with ` too!

File: html-css/module-2/step3-instructions.md (hunk 4 above)
Comment (lareplog) [reply html-css/module-2/step3-instructions.md]: done!!

**Pull request merged**